flybywiresim/aircraft
# fix(a380x/jetway): Jetway connections tweak

Updated interactive_point to tweak Jetway connection to main deck door L1 (better fit) and L2 (jetway fully covers opened door)

Fixes #9366 

Summary of Changes
Updated interactive_point.0 and interactive_point.2 in flight_model.cfg for better jetway connection, with changelog this time.

Main deck L1 - better jetway fit
Main deck L2 - jetway cover opened door fully
(Note: Some jetways are smaller in size and would not cover the opened doors properly! The tweak here is confirmed to work when using Inibuild EGLL jetways)

Screenshots (if necessary)
![Microsoft Flight Simulator Screenshot 2024 11 05 - 21 03 26 84](https://github.com/user-attachments/assets/85a684d5-2b15-4cea-9509-3a3f28bb801a)
![Microsoft Flight Simulator Screenshot 2024 11 05 - 21 04 26 61](https://github.com/user-attachments/assets/e63b5cfc-8a07-4fab-9bcf-1df8d058616c)

References
Additional context
Discord username (if different from GitHub): C.Schaffhausen

Testing instructions
Load into the sim and dock jetways to the aircraft to test both doors
How to download the PR for QA
Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

Make sure you are signed in to GitHub
Click on the Checks tab on the PR
On the left side, find and click on the PR Build tab
Click on either flybywire-aircraft-a320-neo or flybywire-aircraft-a380-842 download link at the bottom of the page
